src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -771,22 +771,27 @@ namespace FourPipeBeam {
 				this->vDotDesignHW = this->vDotNormRatedHW * this->totBeamLength;
 			}
 		}
-		// fill in mass flow rate versions of working variables (regardless of autosizing)
+		// fill in mass flow rate versions of working variables (regardless of autosizing )
 		this->mDotDesignPrimAir = this->vDotDesignPrimAir * DataEnvironment::StdRhoAir;
 
 		if ( ( originalTermUnitSizeMaxVDot > 0.0 ) && ( originalTermUnitSizeMaxVDot != this->vDotDesignPrimAir ) && ( CurZoneEqNum > 0 ) ) {
-			// perturb system size to handle chnage in system size calculated without knowing about 4 pipe beam
-			DataSizing::FinalSysSizing( DataZoneEquipment::ZoneEquipConfig(CurZoneEqNum).AirLoopNum ).DesMainVolFlow
-				+= ( this->vDotDesignPrimAir - originalTermUnitSizeMaxVDot );
-			DataSizing::FinalSysSizing( DataZoneEquipment::ZoneEquipConfig(CurZoneEqNum).AirLoopNum ).DesCoolVolFlow
-				+= ( this->vDotDesignPrimAir - originalTermUnitSizeCoolVDot );
-			DataSizing::FinalSysSizing( DataZoneEquipment::ZoneEquipConfig(CurZoneEqNum).AirLoopNum ).DesHeatVolFlow
-				+= ( this->vDotDesignPrimAir - originalTermUnitSizeHeatVDot );
-			DataSizing::FinalSysSizing( DataZoneEquipment::ZoneEquipConfig(CurZoneEqNum).AirLoopNum ).MassFlowAtCoolPeak
-				+= ( this->vDotDesignPrimAir - originalTermUnitSizeCoolVDot ) * DataEnvironment::StdRhoAir;
-
-			ReportSizingOutput( this->unitType, this->name, "AirLoopHVAC Design Supply Air Flow Rate Adjustment [m3/s]",
-								( this->vDotDesignPrimAir - originalTermUnitSizeMaxVDot ) );
+			if ( DataSizing::FinalSysSizing.size() > 0 ) {
```

Comment:
@mitchute @Myoldmopar @EnergyArchmage I would think that checking SysSizingRunDone should be adequate.  

But there's really no way to use this component without system sizing?  Is that clear in the docs?  Is there a way around it using some assumptions?  This is the only component like that (I think).